### PR TITLE
Fix minor spelling mistakes, add commatas.

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

**Review by RomuloOliveira — APPROVED** [review]
LGTM, @Ana06 wanna take a look?

File: README_de-DE.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Auch wenn das eine Übersetzung darstellen soll, empfiehlt es sich, alle Commit-
 
 ## Was ist ein "Commit"?
 
-Ganz einfach beschrieben ist ein Commit eine Art _Schnappschuss_ deiner lokalen Dateien in deinem Repository (dein Projektordner).
-Im Gegensatz zur geläufigen Annahme [speichert Git nicht nur die Unterschiede in den Dateien sonder immer die gesamte Datei (Quelle auf Englisch)](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences).
-Für Dateien die sich von einem auf den anderen Commit nicht geändert haben, speichert Git lediglich einen Link zur einer bereits gespeicherten Version in einem vorangegangenen Commit.
+Ganz einfach beschrieben, ist ein Commit eine Art _Schnappschuss_ deiner lokalen Dateien in deinem Repository (dein Projektordner).
```
Comment (Ana06) [inline README_de-DE.md]: I think this comma shouldn't be here :confused: If it sounds too weird, I would rather remove the "Ganz einfach beschrieben"

File: README_de-DE.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ Die folgende Grafik zeigt wie Git die Daten mit der Zeit speichert. Jede "Versio
...
-- Um zukünftigen Entwicklern zu helfen zu verstehen wie und warum Änderungen gemacht wurden – was die Fehlersuche und -behebung vereinfacht
+- Um _die_ Gründe zu erläutern, die so nicht aus dem Code hervorgehen (Grundsatzentscheidungen)
+- Um zukünftigen Entwicklern zu helfen, zu verstehen, wie und warum Änderungen gemacht wurden – was die Fehlersuche und -behebung vereinfacht
 
 Um diese und noch mehr Vorteile möglichst effizient nutzen zu können, sollten wir uns an die folgenden Praktiken und Standards halten.
 
 ## Praktiken
 
-Dies sind Praktiken aus meiner eigenen Erfahrung sowie aus anderen Artikeln/Guides aus dem Internet. Wenn du noch mehr Tipps hast oder manche dieser Praktiken für nicht sinnvoll hältst, beantrage gerne eine Änderung via Pull Request.
+Dies sind Praktiken aus meiner eigenen Erfahrung sowie aus anderen Artikeln/Anleitungen aus dem Internet. Wenn du noch mehr Tipps hast oder manche dieser Praktiken für nicht sinnvoll hälst, beantrage gerne eine Änderung via Pull Request.
```
Comment (Ana06) [inline README_de-DE.md]: According to Duden, `hältst` is correct: https://www.duden.de/rechtschreibung/halten

File: README_de-DE.md [hunk 3]
```diff
@@ -259,30 +259,30 @@ Ich persönlich bevorzuge rebase. Gründe dafür sind unter anderem:
...
 
 - dem Reduzieren von kleinen/kontextlosen Commits (Rechtschreibfehler, Formatierung, vergessene Änderungen)
-- dem Zusammenführen mehrere Commits die als Ganzes mehr Sinn ergeben
+- dem Zusammenführen mehrerer Commits, die als Ganzes mehr Sinn ergeben
 - dem Neu-Verpacken von _work in progress_ Commits
 
 ### Wann sollte man rebase bzw. squash vermeiden
 
-Vermeiden Sie die Verwendung von Rebase und Squash in öffentlichen Commits oder in Branches, in denen mehrere Personen arbeiten.
+Vermeiden sie die Verwendung von rebase und squash in öffentlichen Commits oder in Branches, in denen mehrere Personen arbeiten.
```
Comment (Ana06) [inline README_de-DE.md]: I would write `Sie`. It seems that Duden also: https://www.duden.de/sprachwissen/sprachratgeber/Bildung-des-Imperativs 

Ir `Rebase` and `Squash` are names they should be written big. If they are commands, then I would use ``

File: README_de-DE.md [hunk 4]
```diff
@@ -259,30 +259,30 @@ Ich persönlich bevorzuge rebase. Gründe dafür sind unter anderem:
...
 
-Rebase und Squash verändern die Historie der Commits und überschreiben bestehende Commits. Verwendet man rebase und squash auf Commits, die sich auf einen gemeinsam genutzten Branch befinden (d.h. Commits, die in ein Remote Repository verschoben wurden oder von anderen Branches stammen), erzeugt man unötige Verwirrung. Darüber hinaus besteht die Gefahr, dass Andere ihre Änderungen (sowohl lokal als auch auf dem Remote) verlieren, aufgrund von unterschiedlichen Trees und anderen Konflikten.
+rebase und squash verändern die Historie der Commits und überschreiben bestehende Commits. Verwendet man rebase und squash auf Commits, die sich auf einen gemeinsam genutzten Branch befinden (d.h. Commits, die in ein Remote-Repository verschoben wurden oder von anderen Branches stammen), erzeugt man unötige Verwirrung. Darüber hinaus besteht die Gefahr, dass Andere ihre Änderungen (sowohl lokal als auch auf dem Remote) verlieren, aufgrund von unterschiedlichen Trees und anderen Konflikten.
```
Comment (Ana06) [inline README_de-DE.md]: same here

**Review by Ana06 — CHANGES_REQUESTED** [review]
@johnjohndoe thanks for the fixes. I have added a few comments, the rest LGTM. :wink: 

I would write commas instead of commatas in the commit message.

File: README_de-DE.md (hunk 1 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Resolved in 398f8df18e3da8f3506194fe115dec02e84f9b19.

File: README_de-DE.md (hunk 2 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Resolved in 398f8df18e3da8f3506194fe115dec02e84f9b19.

File: README_de-DE.md (hunk 3 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Resolved in 398f8df18e3da8f3506194fe115dec02e84f9b19.

File: README_de-DE.md (hunk 4 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Resolved in 398f8df18e3da8f3506194fe115dec02e84f9b19.

**Review by Ana06 — APPROVED** [review]

File: README_de-DE.md [hunk 5]
```diff
@@ -2,19 +2,19 @@
 
 [![Sag danke!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
 
-Ein Leitfaden um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
+Ein Leitfaden, um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
 
-Verstehe was ein Commit ist, warum es wichtig ist eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
+Verstehe was ein Commit ist, warum es wichtig ist, eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps, um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
```
Comment (bueltge) [inline README_de-DE.md]: maybe 'Bewährte Praktiken' instead of 'Best-Practices'

File: README_de-DE.md [hunk 6]
```diff
@@ -2,19 +2,19 @@
...
 
-Auch wenn das eine Übersetzung darstellen soll, empfiehlt es sich, alle Commit-Messages auf Englisch zu schreiben, deshalb sind die Beispiele nicht übersetzt.
+Auch wenn dies eine Übersetzung darstellen soll, empfiehlt es sich, alle Commit-Messages auf Englisch zu schreiben, deshalb sind die Beispiele nicht übersetzt.
 
 ## Was ist ein "Commit"?
 
 Ganz einfach beschrieben ist ein Commit eine Art _Schnappschuss_ deiner lokalen Dateien in deinem Repository (dein Projektordner).
-Im Gegensatz zur geläufigen Annahme [speichert Git nicht nur die Unterschiede in den Dateien sonder immer die gesamte Datei (Quelle auf Englisch)](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences).
-Für Dateien die sich von einem auf den anderen Commit nicht geändert haben, speichert Git lediglich einen Link zur einer bereits gespeicherten Version in einem vorangegangenen Commit.
+Im Gegensatz zur geläufigen Annahme [speichert Git nicht nur die Unterschiede in den Dateien sondern immer die gesamte Datei (Quelle auf Englisch)](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences).
```
Comment (bueltge) [inline README_de-DE.md]: comma after 'Dateien'

File: README_de-DE.md [hunk 7]
```diff
@@ -24,14 +24,14 @@ Die folgende Grafik zeigt wie Git die Daten mit der Zeit speichert. Jede "Versio
...
-- Um zukünftigen Entwicklern zu helfen zu verstehen wie und warum Änderungen gemacht wurden – was die Fehlersuche und -behebung vereinfacht
+- Um _die_ Gründe zu erläutern, die so nicht aus dem Code hervorgehen (Grundsatzentscheidungen)
+- Um zukünftigen Entwicklern zu helfen, zu verstehen, wie und warum Änderungen gemacht wurden – was die Fehlersuche und -behebung vereinfacht
 
 Um diese und noch mehr Vorteile möglichst effizient nutzen zu können, sollten wir uns an die folgenden Praktiken und Standards halten.
 
 ## Praktiken
 
-Dies sind Praktiken aus meiner eigenen Erfahrung sowie aus anderen Artikeln/Guides aus dem Internet. Wenn du noch mehr Tipps hast oder manche dieser Praktiken für nicht sinnvoll hältst, beantrage gerne eine Änderung via Pull Request.
+Dies sind Praktiken aus meiner eigenen Erfahrung sowie aus anderen Artikeln/Anleitungen aus dem Internet. Wenn du noch mehr Tipps hast oder manche dieser Praktiken für nicht sinnvoll hältst, beantrage gerne eine Änderung via Pull Request.
```
Comment (bueltge) [inline README_de-DE.md]: Comma after 'Erfahrung'

File: README_de-DE.md (hunk 5 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Maybe `gute Praxis`? See d43e636e8118c1660f3e3d83a6624e31a502a80c.

File: README_de-DE.md (hunk 7 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Fixed in d43e636e8118c1660f3e3d83a6624e31a502a80c.

File: README_de-DE.md (hunk 6 above)
Comment (johnjohndoe) [reply README_de-DE.md]: Fixed in d43e636e8118c1660f3e3d83a6624e31a502a80c.

**Comment by johnjohndoe** [comment]
@bueltge @Ana06 @RomuloOliveira I updated and rebased the branch.

**Comment by RomuloOliveira** [comment]
Thank you all!

File: README_de-DE.md (hunk 5 above)
Comment (bueltge) [reply README_de-DE.md]: Yes, also fine; much better as the words in English

**Pull request merged**